etc/api/angular_devkit/core/src/_golden-api.d.ts

```diff
@@ -1018,6 +1018,13 @@ export declare class Workspace {
     loadWorkspaceFromHost(workspacePath: Path): Observable<this>;
     loadWorkspaceFromJson(json: {}): Observable<this>;
     validateAgainstSchema<T = {}>(contentJson: {}, schemaJson: JsonObject): Observable<T>;
+    protected static _workspaceFile: string[];
```

Comment:
`_workspaceFile` looks like it should be `_workspaceFiles` since it is an array.

Reply:
Renamed to `_workspaceFileNames`.